BUILD

```diff
@@ -9,9 +9,15 @@ suitable build configuration is provided and will be used automatically if no ma
 is supplied - skip directly to running "make" (more details below) if you are on one of these
 systems and are happy to use the default configuration.
 
-For other systems, or to fine tune or correct the configuration, create and edit the
```

Comment:
Don't remove the "edit mconfig" option in favour of running the script. Fine tuning or correcting the configuration is still done by editing mconfig, but one way of editing mconfig is running the script. I.e. it should be clear what running the configure script does: it generates an mconfig for you. You should add the information about the configure script after the existing text, not replace the existing text.

Reply:
Actually - you have changed "make" so that it generates a configuration using the configure script if the platform isn't recognised, so the documentation changes here need to reflect that too.

Reply:
I do first part but I need to think about second part.